routes/root.js

```diff
@@ -185,7 +184,7 @@ router.get('/dashboard', async (req, res) => {
     return res.render('dashboard', {
       reactHTML:
         NODE_ENV === 'production'
-          ? await ReactDOMServer.renderToString(React.createElement(DashboardPage, reactProps))
+          ? ReactDOMServer.renderToString(React.createElement(DashboardPage, reactProps))
```

Comment:
This breaks server-side rendering. Please revert.

Reply:
My static analysis says this isn't an async function so await shouldn't do anything here.